Makefile

```
@@ -11,9 +11,32 @@ endif
 PKG      := $(shell go list ./version)
 GIT_INFO := $(shell govvv -flags -pkg $(PKG))
 
+
+ifneq "$(findstring cilium, $(MAKECMDGOALS))" ""
+BUILD_TAG := -tags cilium
+endif
+
+ifneq "$(findstring kubearmor, $(MAKECMDGOALS))" ""
+BUILD_TAG := -tags kubearmor
+endif
+
+ifneq "$(findstring insight, $(MAKECMDGOALS))" ""
+BUILD_TAG := -tags insight
+endif
+
 .PHONY: build
 build:
-	cd $(CURDIR); go mod tidy; CGO_ENABLED=0 go build -ldflags "-w -s ${GIT_INFO}" -o karmor
+	cd $(CURDIR); go mod tidy; CGO_ENABLED=0 go build -ldflags "-w -s ${GIT_INFO}" -tags "cilium kubearmor insight" -o karmor
```

```suggestion
	cd $(CURDIR); go mod tidy; CGO_ENABLED=0 go build -ldflags "-w -s ${GIT_INFO}" -tags "cilium insight" -o karmor
```
I think we won't need a tag to build `kubearmor` related stuff since we should include them by  default.

Also not sure if we support build tags as of now, since the package doesn't seem to be having em

---

No, we don't support the build tags yet, I will remove these changes from this PR and raise a separate PR for that.